File: android/app/src/main/kotlin/com/goalmaker/app/domain/design/ThemeShapes.kt
package com.goalmaker.app.domain.design

/** Corner radii in dp; [FULLY_ROUND] or more means a pill or a circle. */
data class ThemeShapes(
    val card: Int,
    val row: Int,
    val checkbox: Int,
    val button: Int,
) {
    companion object {
        const val FULLY_ROUND = 999
    }
}

File: windows/src/GoalMaker.Core/Design/ThemeShapes.cs
namespace GoalMaker.Core.Design;

/// <summary>Corner radii in device-independent pixels; <see cref="FullyRound"/> or more means a pill or a circle.</summary>
public sealed record ThemeShapes(int Card, int Row, int Checkbox, int Button)
{
    public const int FullyRound = 999;
}
